src/main/java/frc/robot/subsystems/hang/docs/Hang.md

```diff
@@ -0,0 +1,51 @@
+# Hang subsystem
```

Comment:
NOTE: move this into a `docs` folder at the same level as the `src` folder. (see: [RoboLancers 321 Repository](https://github.com/BananasAmIRite/321-Crescendo-2024))